src/notebooks/search_similar_audios.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1e091bae-5d04-42a3-ac01-0982ff289fca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "1c04f4af-7a6a-47e4-9dd9-218749c39ecf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sys.path.append(os.path.abspath('../../app'))"
```

Comment:
@plon-Susk7 why do we have to go back to the `/app` folder? I thought inside docker, we are already in it

Reply:
We' re inside the notebooks folder.

Reply:
I see, got it!